axe.d.ts

```diff
@@ -81,12 +81,28 @@ declare namespace axe {
       };
   type ElementContext = Selector | SelectorList | ContextObject;
 
-  interface SerialContextObject {
+  type SerialSelector =
+    | BaseSelector
+    | LabelledShadowDomSelector
+    | LabelledFramesSelector;
+  type SerialSelectorList = Array<SerialSelector | FramesSelector>;
```

Comment:
While we're here, it'd be helpful to have the internal type pulled out so it can be used directly. That way it's controlled by axe-core rather than duplicated across our apis, so if we update what `SerialSelectorList` is we don't have to update the type that `include` and `exclude` takes throughout all our apis.

Something like this (not sure on the name though)

```suggestion
  type SerialFrameSelector = SerialSelector | FramesSelector;
  type SerialSelectorList = Array<SerialFrameSelector>;
```